src/main/java/frc/robot/commands/AutonomousDelayedDriveStraightCommand.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.SequentialCommandGroupExtended;
+import frc.robot.subsystems.DriveSubsystem;
+
+public class AutonomousDelayedDriveStraightCommand extends SequentialCommandGroupExtended {
+    private final static double DRIVE_SPEED = 0.5;
+    private final static double DRIVE_DISTANCE = 1;
+
+    /** Creates a new AutonomousDelayedDriveStraightCommand. */
+    public AutonomousDelayedDriveStraightCommand(DriveSubsystem driveSubsystem) {
+
+        // Autonomous commands in running order
+        SmartDashboard.putNumber("AutoWaitTime", 0);
```

Comment:
Maybe we could turn `"AutoWaitTime"` into a `private final static` variable? Just to eliminate the chance of accidentally changing one occurrence later and not the other.